apis/grpc/v1/agent/core/agent.pb.go

```diff
@@ -17,7 +17,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.27.1
-// 	protoc        v3.17.3
+// 	protoc        v3.19.1
```

Comment:
In Vald, we are currently using v3.19.1.
https://github.com/vdaas/vald/blob/54691faa35c43ac6e76a114b6f5554d7da9394df/Makefile#L66